VMBackup/main/freezesnapshotter.py

```diff
@@ -378,3 +394,33 @@ def takeSnapshotFromOnlyHost(self):
             self.logger.log('T:S snapshotall ends...', True)
 
         return run_result, run_status, blob_snapshot_info_array, all_failed, unable_to_sleep, is_inconsistent
+
+    def is_command_timedout(self, para_parser):
```

Comment:
to be on safer side , better have everything under try-catch so that it wont break

Reply:
Added